fine-focus.csl

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="chicago" demote-non-dropping-particle="never" default-locale="en-US">
+  <info>
+    <title>Fine Focus</title>
+    <id>http://www.zotero.org/styles/fine-focus</id>
+    <link href="http://www.zotero.org/styles/fine-focus" rel="self"/>
+    <link href="http://www.zotero.org/styles/annual-reviews" rel="template"/>
+    <link href="http://finefocus.org/submissions" rel="documentation"/>
+    <author>
+      <name>Jennifer Huddleston</name>
+      <email>jrh11a@acu.edu</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
```

Comment:
Yes. What is the change it? I changed it in my original file. Where do I need to resubmit that file?